Auth-Page/login.css

```diff
@@ -0,0 +1,246 @@
+/* config.css */
+
+/* helpers/align.css */
+
+.align {
+    -webkit-box-align: center;
+        -ms-flex-align: center;
+            align-items: center;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+        -ms-flex-direction: column;
+            flex-direction: column;
+    -webkit-box-pack: center;
+        -ms-flex-pack: center;
+            justify-content: center;
+  }
+img{
+    width: 65%;
+    margin-left: 2.8rem;
+}
+  .hero-logo{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+    margin-right: 3rem;
+    margin-bottom: 2rem;
+ }
+ .logo-mg{
+     width: 35%;
+     margin-right: -1rem;
+ }
+ .bg-color{
+     color: azure;
+     font-size: 1rem;
+     font-family: monospace;
+     font-weight: 600;
+     margin-right: 2rem;
+ }
+ .small-txt{
+    display: flex;
+    font-size: 13px;
+    flex-flow: row-reverse;
+    color: antiquewhite;
+}
+.pd{
+    padding: 1rem 7rem;
+    color: azure;
+}
+.login-box{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+  
+  /* helpers/grid.css */
+  
+  .grid {
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 320px;
+    max-width: 20rem;
+    width: 90%;
+  }
+  
+  /* helpers/hidden.css */
+  
+  .hidden {
+    border: 0;
+    clip: rect(0 0 0 0);
+    height: 1px;
+    margin: -1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    width: 1px;
+  }
+  
+  /* helpers/icon.css */
+  
+  .icons {
+    display: none;
+  }
+  
+  .icon {
+    display: inline-block;
+    fill: #606468;
+    font-size: 16px;
+    font-size: 1rem;
+    height: 1em;
+    vertical-align: middle;
+    width: 1em;
+  }
+  
+  /* layout/base.css */
+  
+  * {
+    -webkit-box-sizing: inherit;
+    box-sizing: inherit;
+  }
+  
+  html {
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+    font-size: 100%;
+    height: 100%;
+  }
+  
+  body {
+    display: flex;
+    justify-content: center;
+    background-color: #2c3338;
+    color: #606468;
+    font-family: 'Open Sans', sans-serif;
+    font-size: 14px;
+    font-size: 0.875rem;
+    font-weight: 400;
```

Comment:
also use rem for font size,and again you can delete the previous font size if using the second for body